tutorials/experiments/langchain_email_extraction.ipynb

```diff
@@ -1,269 +1,2445 @@
 {
```

Comment:
<p>We're using the term "evaluator" rather than "evaluation". Might also refer to the dataset as a "collection" rather than a list to avoid confusion with the Python data type.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3805/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>